django/core/files/storage/base.py

```diff
@@ -104,13 +104,9 @@ def generate_filename(self, filename):
         Validate the filename by calling get_valid_name() and return a filename
         to be passed to the save() method.
         """
-        filename = str(filename).replace("\\", "/")
```

Comment:
Visually, this looks like the matching change but in a different commit. 🤔

Reply:
The "narrative" behind this change is "substitution of equals for equals". Given `validate_file _name` which is, in the previous commit, tested to be correct and complete, replace the logic in this method with an equivalent from the helper.